core/ir/instr_create_shared_api.h

```diff
@@ -54,6 +54,7 @@
 #    include "dr_ir_instr.h"
 #    include "dr_ir_utils.h"
 #endif
+#include <limits.h> /* For SCHAR_MAX, SCHAR_MIN. */
```

Comment:
Are `SCHAR_MAX` and `SCHAR_MIN` supposed to be added by this PR?

Reply:
They are already present below in OPND_CREATE_INT_32OR8.  I was using a mov immed to set the regs to 0 originally and hit the missing include problem.  Later I changed the mov immed to a sub as it was simpler than getting the immed size right but left this fix; maybe could separate it but not sure it's worth the effort.

Reply:
We can surely keep it if it required. I was just curious why it is still needed (as the mov immed that prompted this isn't there anymore).